packages/@aws-cdk/aws-ec2/test/connections.test.ts

```diff
@@ -10,7 +10,7 @@ import {
   Vpc,
 } from '../lib';
 
-export = {
+nodeunitShim({
```

Comment:
Nit (don’t change anything!): I expected this to be called `jestShim` given it’s part of the nodeunit-shim module.

Reply:
But thank god for this!